<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IndividualSetUI</class>
 <widget class="QWidget" name="IndividualSetUI">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QFrame" name="frameIndividualSettings">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QCheckBox" name="cbEnableIndividualConsign">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Utiliser les paramètres définis ci-dessous pour cet ensemble</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget {
  
	color : rgb(0,106,255);
    font:  bold 12pt Calibri;


 }
</string>
        </property>
        <property name="text">
         <string>Utiliser les paramètres individuels</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbAllIndividualParameters">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Utiliser les paramètres individuels pour tous les ensembles</string>
        </property>
        <property name="styleSheet">
         <string notr="true">

QPushButton {
    font: bold 12pt Calibri;  
    border: 1px solid gray;
    border-radius: 3px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
 }
 
QPushButton::enabled {
	color : rgb(0,106,255);
    border: 1px solid rgb(0,106,255);
 }

QPushButton:hover:!pressed{

     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
}

 QPushButton:pressed{
    background-color:  rgb(220,220,220);

}</string>
        </property>
        <property name="text">
         <string>I</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbAllGlobalParameters">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Utiliser les paramètres globaux pour tous les ensembles</string>
        </property>
        <property name="styleSheet">
         <string notr="true">

QPushButton {
    font: bold 12pt Calibri;  
    border: 1px solid gray;
    border-radius: 3px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
 }
 
QPushButton::enabled {
	color : rgb(0,106,255);
    border: 1px solid rgb(0,106,255);
 }

QPushButton:hover:!pressed{

     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
}

 QPushButton:pressed{
    background-color:  rgb(220,220,220);

}</string>
        </property>
        <property name="text">
         <string>G</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbCopyGlobalToIndividuals">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Initialiser tous les ensembles individuels avec le paramètre global</string>
        </property>
        <property name="styleSheet">
         <string notr="true">

QPushButton {
    font: bold 12pt Calibri;  
    border: 1px solid gray;
    border-radius: 3px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
 }
 
QPushButton::enabled {
	color : rgb(0,106,255);
    border: 1px solid rgb(0,106,255);
 }

QPushButton:hover:!pressed{

     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
}

 QPushButton:pressed{
    background-color:  rgb(220,220,220);

}</string>
        </property>
        <property name="text">
         <string>I=G</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbCopyIndividualToGlobal">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Initialiser le paramètre global avec le paramètre individuel courant</string>
        </property>
        <property name="styleSheet">
         <string notr="true">

QPushButton {
    font: bold 12pt Calibri;  
    border: 1px solid gray;
    border-radius: 3px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
 }
 
QPushButton::enabled {
	color : rgb(0,106,255);
    border: 1px solid rgb(0,106,255);
 }

QPushButton:hover:!pressed{

     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
}

 QPushButton:pressed{
    background-color:  rgb(220,220,220);

}</string>
        </property>
        <property name="text">
         <string>G=I</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameSet">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>500</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTabWidget" name="tabSet">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">

QTabWidget::tab-bar {
    alignment: center;
}

QTabWidget::pane { 

	/* border */
    border: 0px solid rgb(0,106,255);
    border-top: 1px solid rgb(0,106,255);
	border-right: 0px solid rgb(0,106,255);
	border-bottom: 1px solid rgb(0,106,255);

	/* border radius */
	border-top-right-radius: 0px;
	border-top-left-radius: 0px;
	border-bottom-right-radius: 0px;
	border-bottom-left-radius: 0px;

	/* color */
	background-color:rgb(255, 255, 255)
}

QTabBar::tab {

	/* border */
    border: 0px solid rgb(0,106,255);
    border-right: 1px solid rgb(0,106,255);
    border-top: 1px solid rgb(0,106,255);

	/* border radius */
    border-radius: 0px;

	/* color */
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
	color : rgb(0,106,255);

	/* size */
	height:4ex;
	min-width:20ex;

	/* padding */
    padding: 2px;
}

/* ################ SELECTION */

QTabBar::tab:!selected {

	/* color */
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
	color : rgb(127,180,255);

	/* font */
  	font:  11pt Calibri;
}


QTabBar::tab:selected {

	/* color */
	color : rgb(0,106,255);
    background-color:  rgb(220,220,220);

	/* font */
	 font: bold 11pt Calibri;
}


/* ################ HOVER*/

QTabBar::tab:hover  {

	/* color */
	color : rgb(0,106,255);	
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);

	/* font */
	 font: bold 11pt Calibri;
}


/* ################ ORDER */




QTabBar::tab:first{

	/* border */
	border-left: 1px solid rgb(0,106,255);

	/* border radius */
	border-top-left-radius: 3px;
	border-bottom-left-radius: 0px;

}

QTabBar::tab:last{

	/* border radius */
	 border-top-right-radius: 4px;
}

QTabBar::tab:!enabled{
	color: rgb(150,150,150);
}



QTabWidget::pane:!enabled { 
/*	border-color : rgb(150,150,150);*/
	color: rgb(150,150,150);
}


</string>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <property name="elideMode">
         <enum>Qt::ElideLeft</enum>
        </property>
        <widget class="QWidget" name="tabSetText">
         <attribute name="title">
          <string>TEXTE</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tabSetStyle">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <attribute name="title">
          <string>STYLE</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_7">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="vlSetStyle">
            <property name="spacing">
             <number>0</number>
            </property>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
